### PR TITLE
Add support for EVW3200-Wifi + CLI tool

### PR DESCRIPTION
**Pull request by StevenLooman**
I noticed this library via the Home Assistant integration. Great work!

The library, as advertised, has been written for and tested with `Ubee EVW32C-0N`. Unfortunately, this does not work for my `Ubee EVW3200-Wifi` router/cable modem. This pull request adds support for this device.

Please do not merge yet, but do test it on your own router. This works for me, but I might have broken something for your model.

### CODE REVIEW

**Comment by mzdrale** [comment]
Thanks @StevenLooman, great work!
I just installed your version. I will give it a try and I will let you know if there is need to change something. So far looks good.

File: README.md [hunk 1]
```diff
@@ -30,6 +31,30 @@ for x in devices:
...
+```
+$ pyubee --help
+usage: pyubee [-h] [--model MODEL] host username password
+
+pyubee
+
+positional arguments:
+  host           Host
+  username       Username
+  password       Username
```
Comment (mzdrale) [inline README.md]: This is typo, should be `password    Password`. ;)

File: pyubee/__main__.py [hunk 2]
```diff
@@ -1,10 +1,35 @@
...
+from pyubee import SUPPORTED_MODELS
 from pyubee import Ubee
 
+
 def main():
     """Scan for devices and print results."""
+    parser = argparse.ArgumentParser(description='pyubee')
+    parser.add_argument('host', help='Host')
+    parser.add_argument('username', help='Username')
+    parser.add_argument('password', help='Username')
```
Comment (mzdrale) [inline pyubee/__main__.py]: Typo. Should be:

`parser.add_argument('password', help='Password')`

**Review by mzdrale — CHANGES_REQUESTED** [review]
So far looks good. There is only one typo in code and README.

**Comment by StevenLooman** [comment]
Thank you for the review, fixed!

I think you've checked the columns names as well. I'm removing the question marks.

**Review by mzdrale — APPROVED** [review]

**Comment by mzdrale** [comment]
Thank you @StevenLooman!
I've merged your changes and released version 0.3 ([GitHub](https://github.com/mzdrale/pyubee/releases), [PyPi](https://pypi.org/project/pyubee/)).

I've also made some changes to [Home Assistant Ubee Device Tracker](https://www.home-assistant.io/components/device_tracker.ubee), but I want to test it more before I create Home Assistant PR.

**Comment by StevenLooman** [comment]
Great! Can I help with testing?

**Comment by mzdrale** [comment]
Sure, it would be great. I've sent email to you with instructions.

**Pull request merged**